core/domain/html_validation_service.py

```diff
@@ -526,9 +526,11 @@ def get_latex_strings_without_svg_from_html(html_string):
         svg_filename = (
             objects.UnicodeString.normalize(math_content_dict['svg_filename']))
         if svg_filename == '':
-            latex_strings.add(raw_latex.encode('utf-8'))
+            # The raw_latex is string, but we want to return list of bytes,
+            # so we need to need to encode it.
```

Comment:
Duplicate "need to".

Why do we want to return a list of bytes, btw? Don't we typically prefer to pass around unicode strings by default?

Reply:
I just discovered that we don't need this function at all. Removed.